people/ioana-chiorean

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: QA Team Lead, Open Source Evanghelist, Open Designer Enthusiast 
```

Comment:
It could be a try to mislead search engines, but evangelist is written wrong -- both times.